packages/core/src/Site/index.ts

```diff
@@ -914,18 +983,14 @@ class Site {
         || !_.isEqual(oldSiteConfig.plantumlCheck, this.siteConfig.plantumlCheck);
 
     if (isGlobalConfigModified() || !_.isEmpty(addedPages) || !_.isEmpty(removedPages)) {
-      await this.removeAsset(removedPages);
+      this.removeAsset(removedPages);
```

Comment:
Do we not need to await this?

Reply:
Ah you're right, we need to await it. Added `async` to the 3 method signatures to reflect the asynchronous status!